packages/ui/src/components/form/field.tsx

```diff
@@ -30,7 +31,7 @@ export const Field = forwardRef(
     return (
       <RadixForm.Field name={name} ref={reference} className={className}>
         {label && (
-          <RadixForm.Label className="mb-2 block text-label4 text-neutral-700">
+          <RadixForm.Label className="mb-2 block text-label4 text-secondary">
```

Comment:
do we want to change this color? 
![image](https://github.com/user-attachments/assets/31486a6b-376a-4ec2-8617-cc97c4d65d65)

Reply:
it should be neutralGreen-700
neutral-700 it's #5F616E

Reply:
you're right, let's change it to `naturalGreen-700`

Reply:
changed